stac_fastapi/types/stac_fastapi/types/transaction.py

```diff
@@ -0,0 +1,133 @@
+"""Transaction extension types."""
```

Comment:
can we move back everything in `stac_fastapi.extensions`? 

by importing `CollectionUri, ItemUri` from `stac_fastapi.api` you create a circular dependency between `types` and `api`. `api` depends on `types` and `types` shouldn't depend on `api`